src/Tests/RemoteUnitTestExecutor/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace RemoteUnitTestExecutor
+{
+    public class Program
+    {
+        [LoaderOptimization(LoaderOptimization.MultiDomainHost)]
```

Comment:
>[LoaderOptimization(LoaderOptimization.MultiDomainHost)] [](http://example.com/codeflow?start=8&length=56)

This is interesting, and only meaningful in .NET framework. Why is this here?

Reply:
The RemoteUnitTestExecutor is only for .NET Framework tests. This assembly and the test assemblies that depend on it build for .NET Framework 4.5. Why this is here, I'm not certain, but its in the original code.